### PR TITLE
Raise error with better information and adding german translation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by nbessi** [comment]
Ok for the fix, I relaunnch travis let see if it is green

**Comment by nbessi** [comment]
Travis restarted

**Comment by nbessi** [comment]
I can not find faulty manifest for Direct Debit

File: l10n_ch_base_bank/invoice.py [hunk 1]
```diff
@@ -19,6 +19,8 @@
 ##############################################################################
 from openerp.osv.orm import Model
 from openerp.tools import mod10r
+from openerp.osv import fields, osv
```
Comment (yvaucher) [inline l10n_ch_base_bank/invoice.py]:
```
./l10n_ch_base_bank/invoice.py:22:1: F401 'fields' imported but unused
```

Could you remove fields import, it fails on pep8 check on travis

**Comment by ghost** [comment]
I update the import and the language file.

**Comment by nbessi** [comment]
:+1:  @BT-csanchez have you signed the OCA cla?

**Comment by gurneyalex** [comment]
@nbessi @BT-csanchez  is covered by his employer's ECLA. I updated the OCA database.

**Comment by tafaRU** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**